docs/content/guides/developer/advanced/custom-indexer.mdx

```diff
@@ -41,26 +41,6 @@ $ psql --version
 $ diesel --version
 ```
 
-To test PostgreSQL functionality, you can create and delete a test table.
```

Comment:
I left this section here on purpose. The point is to test functionality before they go any further. But ultimately not a hill I would die on.